src/AI/detection_model/yolo.py

```diff
@@ -117,14 +124,12 @@ def delete_images(path):
         elif os.path.isdir(item_path):
             # Remove directories and their contents (recursively)
             shutil.rmtree(item_path)
-            
-            
-def run_classification(_image):
 
-    predict_image(_image, rotations=0)
-    _images = crop_images(path="boxes/", target_size=(224, 224))
-
-    if len(_images) > 0:
-        return _images[0]
+def run_classification(_image):
+    prediction_results = predict_image(_image, rotations=0)     #prediction_results[Bird index][0 = image, 5 = accuracy]
+    _res_image = crop_images(prediction_results[0][0], target_size=(224, 224)) # We send the first bird box image
+    _res_image.show()
+    if _res_image:
+        return _res_image
```

Comment:
Ska bara behöva returna _res_image, antingen är den nil eller inte. Behövs image.show()?
```suggestion
    return _res_image
```

Reply:
removed image show keep if statment for better handling